packages/data/addon/models/metadata/column.ts

```diff
@@ -8,6 +8,15 @@ import { inject as service } from '@ember/service';
 import KegService from '../../services/keg';
 import Table from './table';
 
+export type ColumnNode = {
```

Comment:
Does it make sense to put the raw graphql types in the model file? Seems like more of a serializer thing.